docs/website/docs/plus/features/project/overview.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Project Overview
+description: Generate dlt projects in yaml
+---
+
+# Project
+
+
+<img src="https://storage.googleapis.com/dlt-blog-images/plus/dlt_plus_projects.png" width="500"/>
+
+[dlt+ Project](../../core-concepts/project.md) provides a structured and opinionated approach to organizing data workflows while implementing best practices for data engineering teams. dlt+ Project automates key processes such as data loading, data transformations, data catalogs, and data governance, and enables different members of the data teams to work more easily with each other.
+
+With dlt+ Project, you can efficiently manage your data workflows by:
+
+1. [Using a declarative `dlt.yml` file](#the-dlt-manifest-file-dltyml) to define sources, destinations, pipelines, and transformations.
+2. Configuring [different profiles](../../core-concepts/profiles.md) for various use cases and environments.
+3. Ensuring data quality by defining tests with [dlt+ tests utils](../quality/tests.md).
+4. Packaging your project as a Python package and distributing it via PyPI or a git repository [Coming Soon!]
```

Comment:
It's already there, we have a tutorial: https://deploy-preview-2925--dlt-hub-docs.netlify.app/docs/devel/plus/getting-started/advanced_tutorial